aiodocker/services.py

```diff
@@ -87,11 +87,51 @@ def __init__(self, docker):
         response = await self.docker._query_json(
             "services/create",
             method="POST",
-            headers={"Content-type": "application/json", },
             data=data,
         )
         return response
 
+    async def update(self,
+                     service_id: str,
+                     *,
+                     image: str=None,
+                     rollback: bool=False
+                     ) -> bool:
+        """
+        Update a service.
+        If rollback is True image will be ignored.
+
+        Args:
+            service_id: ID or name of the service.
+            rollback: Rollback the service to the previous service spec.
+
+        Returns:
+            True if successful.
+        """
+        if image is None and rollback is False:
```

Comment:
Do you have any idea why the method is not covered by test?
At least codecov reports it.

Reply:
Resolved. 👍

Reply:
Cool!